Examples/Algorithms/TrackFinding/src/HoughTransformSeeder.cpp

```diff
@@ -281,43 +281,60 @@ ActsExamples::HoughHist ActsExamples::HoughTransformSeeder::createHoughHist(
 bool ActsExamples::HoughTransformSeeder::passThreshold(
     HoughHist const& houghHist, unsigned x, unsigned y) const {
   // Pass window threshold
-  unsigned width = m_cfg.threshold.size() / 2;
+  const unsigned width = m_cfg.threshold.size() / 2;
+
   if (x < width || m_cfg.houghHistSize_x - x < width) {
     return false;
   }
+
   for (unsigned i = 0; i < m_cfg.threshold.size(); i++) {
     if (houghHist.atLocalBins({y, x - width + i}).first < m_cfg.threshold[i]) {
       return false;
     }
   }
 
   // Pass local-maximum check, if used
-  if (m_cfg.localMaxWindowSize != 0) {
-    for (int j = -m_cfg.localMaxWindowSize; j <= m_cfg.localMaxWindowSize;
-         j++) {
-      for (int i = -m_cfg.localMaxWindowSize; i <= m_cfg.localMaxWindowSize;
-           i++) {
-        if (i == 0 && j == 0) {
-          continue;
-        }
-        if (y + j < m_cfg.houghHistSize_y && x + i < m_cfg.houghHistSize_x) {
-          if (houghHist.atLocalBins({y + j, x + i}).first >
-              houghHist.atLocalBins({y, x}).first) {
-            return false;
-          }
-          if (houghHist.atLocalBins({y + j, x + i}).first ==
-              houghHist.atLocalBins({y, x}).first) {
-            if (houghHist.atLocalBins({y + j, x + i}).second.size() >
-                houghHist.atLocalBins({y, x}).second.size()) {
-              return false;
-            }
-            if (houghHist.atLocalBins({y + j, x + i}).second.size() ==
-                    houghHist.atLocalBins({y, x}).second.size() &&
-                j <= 0 && i <= 0) {
-              return false;  // favor bottom-left (low phi, low neg q/pt)
-            }
-          }
-        }
+  if (m_cfg.localMaxWindowSize == 0) {
+    return true;
+  }
+
+  for (int j = -m_cfg.localMaxWindowSize; j <= m_cfg.localMaxWindowSize; j++) {
```

Comment:
Is our test coverage high enough to be able to see if this is equivalent? /cc @goblirsc